Python/Product/Analysis/LanguageServer/Server.SignatureHelp.cs

```diff
@@ -88,34 +88,66 @@ public sealed partial class Server {
         private SignatureInformation ToSignatureInformation(IOverloadResult overload) {
             var si = new SignatureInformation();
 
+            si.parameters = overload.Parameters.MaybeEnumerate().Select(p => new ParameterInformation {
+                label = p.Name,
+                documentation = string.IsNullOrEmpty(p.Documentation) ? null : p.Documentation,
+                _type = p.Type,
+                _defaultValue = p.DefaultValue
+            }).ToArray();
+
+            si._returnTypes = (overload as IOverloadResult2)?.ReturnType.OrderBy(k => k).ToArray();
+
             if (_clientCaps?.textDocument?.signatureHelp?.signatureInformation?._shortLabel ?? false) {
                 si.label = overload.Name;
             } else {
+                var doc = overload.Documentation;
+                // Some function contain signature in the documentation. Example: print.
+                // We want to use that signature in VS Code since it contains all arguments.
+                if (si.parameters.Length == 0 && !string.IsNullOrEmpty(doc) && doc.StartsWithOrdinal($"{overload.Name}(")) {
```

Comment:
Can we have space between `{overload.Name}` and `(`?

Reply:
Normally no since Python is typically formatted to PEP.